added note (#5980)
MicrosoftDocs/windows-itpro-docs: windows/deployment/windows-autopilot/windows-autopilot-requirements.md

- [Intune for Education subscriptions](https://docs.microsoft.com/intune-education/what-is-intune-for-education), which include all needed Azure AD and Intune features.
- [Azure Active Directory Premium P1 or P2](https://azure.microsoft.com/services/active-directory/) and [Microsoft Intune subscriptions](https://www.microsoft.com/cloud-platform/microsoft-intune) (or an alternative MDM service).

> >[!NOTE]

Reviewer: MarkDown indent compatibility spacing:
```suggestion
> [!NOTE]
```

> >In case of using Microsoft 365 subscriptions, you still need to [assign Intune licenses to the users](https://docs.microsoft.com/intune/fundamentals/licenses-assign)

Reviewer: @JohanFreelancer9 : May I request your recommendation for this sentence structure?
```suggestion
> When using Microsoft 365 subscriptions, you still need to [assign Intune licenses to the users](https://docs.microsoft.com/intune/fundamentals/licenses-assign)
```

Reviewer 2: ```suggestion
> Even when using Microsoft 365 subscriptions, you still need to [assign Intune licenses to the users](https://docs.microsoft.com/intune/fundamentals/licenses-assign).
```
